torch/distributed/pipelining/schedules.py

```diff
@@ -1041,6 +1041,77 @@ def __init__(
                 "Simply stop passing it, and everything should still work fine."
             )
 
+    def seeded_module_init(
```

Comment:
#bikeshed_me
"seeded_module_init"?  meh.  
I first used 'initialize_stage' but i realized there is a preexisting '_initialize_stage' API right below that is confusing.